README.md

```diff
@@ -46,6 +48,27 @@ RUSTFLAGS=--cfg=web_sys_unstable_apis cargo build --target wasm32-unknown-unknow
 wasm-bindgen --out-dir target/generated --web target/wasm32-unknown-unknown/debug/examples/hello-triangle.wasm
 ```
 
+Create an `index.html` file into `target/generated` directory and add the following code:
+
+```html
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="google" content="notranslate">
```

Comment:
nit: we could remove this `notranslate` tag and the `title` to make this a bit shorter

Reply:
agreed, can follow up :)

Reply:
Well, I do not really like the Chrome's translation popin when I develop a website. Bad habit ^^'.